src/containers/settings/index.tsx

```diff
@@ -68,15 +90,33 @@ const Settings: React.FC = () => {
               <Form.Item {...formLayout} name="username">
                 <Input placeholder="Username" />
               </Form.Item>
-
-              <FormTitle>Change Email</FormTitle>
+            </Form>
+            <FormTitle>Change Email</FormTitle>
+            <Form
+              {...formLayout}
+              name="change-email"
+              onFinish={onFinishChangeEmail}
+            >
               <Form.Item {...formLayout} name="current-email">
                 <Input
                   placeholder="Current Address"
                   defaultValue="currentemail@email.com"
                 />
               </Form.Item>
-              <Form.Item {...formLayout} name="new-email">
+              <Form.Item
+                {...formLayout}
+                name="new-email"
+                rules={[
+                  {
+                    required: true,
+                    message: 'Please input a new email address!',
+                  },
+                  {
+                    pattern: /^\S+@\S+\.\S{2,}$/,
```

Comment:
This pattern should probably be extracted somewhere/compared against whatever we use on the backend

Reply:
Checked and matches backend